README.md

```diff
@@ -336,10 +365,6 @@ Further details are written at http://docs.fluentd.org/articles/in_exec
 Skip adding geoip fields when this valaues to `true`.
 On the case of getting nothing of GeoIP info (such as local IP), it will output the original record without changing anything.
 
-* `flush_interval` (default: 0 sec)
```

Comment:
This is in filter geoip section.
We must remove about `flush_interval` in this section because filter plugin does not support `flush_interval`.